keadm/cmd/keadm/app/cmd/util/helminstaller.go

```diff
@@ -381,15 +381,9 @@ func (cu *KubeCloudHelmInstTool) handleProfile(profileValue string) error {
 		if profileValue == "" {
 			profileValue = types.HelmDefaultVersion
 		}
-		profileValueSuffix := strings.TrimPrefix(profileValue, "v")
-		// confirm it startswith "v"
-		if profileValue != profileValueSuffix {
-			cu.Sets = append(cu.Sets, fmt.Sprintf("%s=v%s", "cloudCore.image.tag", profileValueSuffix))
-			cu.Sets = append(cu.Sets, fmt.Sprintf("%s=v%s", "iptablesManager.image.tag", profileValueSuffix))
-		} else {
-			cu.Sets = append(cu.Sets, fmt.Sprintf("%s=%s", "cloudCore.image.tag", profileValue))
-			cu.Sets = append(cu.Sets, fmt.Sprintf("%s=%s", "iptablesManager.image.tag", profileValue))
-		}
```

Comment:
v is no need now?

Reply:
Yeah, this is a custom flag,  "cloudCore.image.tag" or "iptablesmgr.image.tag" is controlled by user if they have a custom image. If not given, will use the default kubeedge version.

Reply:
Okey, users can set the custom value, so if we always add the `v`, it not reasonable, right?

Reply:
> Okey, users can set the custom value, so if we always add the `v`, it not reasonable, right?

I think using v is professional, but once the user sets a custom image tag, should override our default version.

Reply:
In k8s/kubeadm, it has a version match, if match successfully, will add  the `v`. If match failed, which should means the version is customed, will not add `v`.

I think users may set the version but ignore the `v` prefix, so deploy will failed, how about we also  have a check like k8s? @zhu733756

Reply:
Ref: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/version.go#L152

Reply:
Seems good.

Reply:
But I am worried that if the user gives a tag that matched the regex but is not given a "v" prefix string,  their custom image tag would not match through this method when we return a "v" prefix.